assets/js/modules/tagmanager/datastore/__factories__/utils.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Tag Manager factory utils.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { STORE_NAME } from '../constants';
+import {
+	buildLiveContainerVersionWeb,
+	buildLiveContainerVersionAMP,
+} from './builders';
+
+export const parseLiveContainerVersionIDs = ( { accountId, containerId, container: { publicId } }, callback ) => {
```

Comment:
This function signature is a bit unwieldy, and I'm wondering about the `Id` vs `ID` capitalisation here versus elsewhere. Having `container` be its own variable and then separating out `publicId` inside the function would make this function's signature much easier to read. I think we should do that.

I think the capitalisation exception is due to the way the data is sent from the API, but in that case: let's explicitly mention it like we do [elsewhere](https://github.com/google/site-kit-wp/blob/develop/assets/js/modules/analytics/components/common/PropertySelect.js#L76).